server/testsupport/curiousAssessmentsTestDataBuilder.ts

```diff
@@ -0,0 +1,469 @@
+  establishmentName: options.prisonName,
+  lddPrimaryName: options.lddPrimaryName,
+  lddSecondaryNames: options.lddSecondaryNames,
+  inDepthAssessmentDate: options.inDepthAssessmentDate,
+  rapidAssessmentDate: options.rapidAssessmentDate,
+})
+
+const aLearnerAssessmentV2DTO = (
+  options: {
+    prisonNumber?: string
+    assessments?: ExternalAssessmentsDTO
+  } = {
+    prisonNumber: 'A1234BC',
+    assessments: anExternalAssessmentsDTO(),
+  },
+): LearnerAssessmentV2DTO => ({
+  prn: options.prisonNumber,
+  assessments: options.assessments,
+})
+
+const anExternalAssessmentsDTO = (
+  options: {
+    alnAssessments?: Array<LearnerAssessmentsAlnDTO>
+    digitalFunctionalSkillsAssessments?: Array<LearnerAssessmentsFunctionalSkillsDTO>
+    englishFunctionalSkills?: Array<LearnerAssessmentsFunctionalSkillsDTO>
+    mathsFunctionalSkills?: Array<LearnerAssessmentsFunctionalSkillsDTO>
+    esolAssessments?: Array<LearnerAssessmentsDTO>
+    readingAssessments?: Array<LearnerAssessmentsDTO>
+  } = {
+    alnAssessments: [anAlnLearnerAssessmentsDTO()],
+    digitalFunctionalSkillsAssessments: [aDigitalFunctionalSkillsLearnerAssessmentsDTO()],
+    englishFunctionalSkills: [anEnglishFunctionalSkillsLearnerAssessmentsDTO()],
+    mathsFunctionalSkills: [aMathsFunctionalSkillsLearnerAssessmentsDTO()],
+    esolAssessments: [anEsolLearnerAssessmentsDTO()],
+    readingAssessments: [aReadingLearnerAssessmentsDTO()],
+  },
+): ExternalAssessmentsDTO => ({
+  aln: options.alnAssessments,
+  digitalSkillsFunctionalSkills: options.digitalFunctionalSkillsAssessments,
+  englishFunctionalSkills: options.englishFunctionalSkills,
+  mathsFunctionalSkills: options.mathsFunctionalSkills,
+  esol: options.esolAssessments,
+  reading: options.readingAssessments,
+})
+
+const anAlnLearnerAssessmentsDTO = (
+  options: {
+    prisonId?: string
+    prisonName?: string
+    assessmentDate?: string
+    assessmentOutcome?: 'Yes' | 'No'
+    hasPrisonerConsent?: 'Yes' | 'No'
+    stakeholderReferral?:
+      | 'Healthcare'
+      | 'Psychology'
+      | 'Education'
+      | 'Specialist'
+      | 'NSM'
+      | 'Substance Misuse Team'
+      | 'Safer Custody'
+      | 'Other'
+  } = {
+    prisonId: 'MDI',
+    prisonName: 'MOORLAND (HMP & YOI)',
+    assessmentDate: '2025-10-01',
+    assessmentOutcome: 'Yes',
+    hasPrisonerConsent: 'Yes',
+    stakeholderReferral: 'Education',
+  },
+): LearnerAssessmentsAlnDTO => ({
+  establishmentId: options.prisonId,
+  establishmentName: options.prisonName,
+  assessmentDate: options.assessmentDate,
+  assessmentOutcome: options.assessmentOutcome,
+  hasPrisonerConsent: options.hasPrisonerConsent,
+  stakeholderReferral: options.stakeholderReferral,
+})
+
+const aReadingLearnerAssessmentsDTO = (
+  options: {
+    prisonId?: string
+    prisonName?: string
+    assessmentDate?: string
+    assessmentNextStep?: 'Refer for reading support level.' | 'Reading support not required at this time.'
```

Comment:
This is the builder for a Reading assessment, type `LearnerAssessmentsDTO`
The following function is the builder for an ESOL assessment, also of type `LearnerAssessmentsDTO`

Whilst they are both of the same type, the possible values in `assessmentNextStep` and `assessmentOutcome` are very very different!